src/styles/01-atoms/links/link/_link.scss

```diff
@@ -5,6 +5,10 @@ $color-link--hover: $nypl-blue-dark;
 @mixin link {
   color: $color-link;
 
+  &--no-underline {
```

Comment:
@helenvholmes I'm gonna tag you in the stuff that I changed that would be good to sanity check.  This is to get that blue text link with no underline (http://themetronome.co/components/mc-25-search-results-item/?fresh=true)

Reply:
Ah, on the header? Willa might actually ask us to change that eventually, now that I'm looking at it...

Code looks good to me, though!